frontend/src/contexts/players.tsx

```diff
@@ -1,14 +1,14 @@
-import React, { useState } from "react";
+import React, {
+  PropsWithChildren,
+  useCallback,
+  useContext,
+  useMemo,
+  useState,
+} from "react";
 import useCrud from "../hooks/useCrud";
+import { Player } from "../types";
```

Comment:
This is a nice change

Reply:
Would like to consolidate these types to a shared definition across the backend & frontend down the track but this is a good step for now